CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [2.2.1] - 2020-11-18
+
+### Changed
+- [`<DropdownShell>`](https://illright.github.io/attractions/docs/components/dropdown-shell): Now closes the dropdown when pressing <kbd>Enter</kbd> or <kbd>Escape</kbd> buttons.
+- [`<TimePicker>`](https://illright.github.io/attractions/docs/components/time-picker): The currently selected value will now be highlighted in the dropdown. Labels are customizable as slots.
+
+(Thanks to [@RikuVan](https://github.com/RikuVan) for these 2 contributions - [#150](https://github.com/illright/attractions/pull/150))
```

Comment:
I'm thinking of maybe boldfacing the name. Will wait until it builds to see if it's worth it

Reply:
Seems like an internal update to GitHub Actions broke our workflow. Will fix in #156. This is how it looks like in the docs:
![image](https://user-images.githubusercontent.com/11016151/99539275-156eb000-29bf-11eb-9084-07d15585efe2.png)

Reply:
Okay, it's not worth it. But I must confess, somehow I don't like this styling. I may do something with it in the future.